### ISSUE TITLE
Error in loqus_extension.py

### ISSUE THREAD

**Issue by northwestwitch** [Clinical-Genomics/scout#2782]
To reproduce go to any variant of a case on scout stage. Error:

```
Traceback (most recent call last):
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 2446, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1951, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1820, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1949, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/site-packages/flask/app.py", line 1935, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/utils.py", line 39, in decorated_function
    context = func(*args, **kwargs)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/variant/views.py", line 53, in variant
    data["observations"] = observations(store, loqusdb, data["case"], data["variant"])
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/blueprints/variant/controllers.py", line 341, in observations
    obs_data = loqusdb.get_variant(variant_query, loqusdb_id=loqusdb_id)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/extensions/loqus_extension.py", line 207, in get_variant
    return self.get_exec_loqus_variant(loqus_instance, variant_info)
  File "/home/hiseq.clinical/SERVER/apps/scout-stage/git/scout-stage/scout/server/extensions/loqus_extension.py", line 284, in get_exec_loqus_variant
    return json.loads(output)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/home/hiseq.clinical/miniconda/envs/prod-stage/lib/python3.6/json/decoder.py", line 342, in decode
    raise JSONDecodeError("Extra data", s, end)
json.decoder.JSONDecodeError: Extra data: line 1 column 5 (char 4)
```

**Comment by mikaell** [comment]
I can have a look at this.

**Comment by northwestwitch** [comment]
This was nicely fixed by #2785, closing the issue!
